src/views/CreatePage.vue

```diff
@@ -63,9 +63,8 @@ div
     option(value="left") Left
     option(value="center") Center
     option(value="right") Right
-div(class="p-5 bg-orange-100")
-  div(ref="stageParentDiv" class="bg-white")
-
+div(class="flex justify-center items-center p-5 bg-slate-200")
+  div(ref="stageParentDiv" class="bg-white max-w-screen-xl w-full")
```

Comment:
```suggestion
div(class="m-auto border-4 max-w-screen-xl")
  div(ref="stageParentDiv" class="bg-white w-full")
```
今1540px以上だと余りが出てしまうので、
cssを直すとgifのようにキレイ？に表示されます。

![画面収録_2022-10-16_23_14_55_AdobeExpress](https://user-images.githubusercontent.com/68400191/196040678-42708bd7-3f06-4e6a-81c1-71648c6b911c.gif)